core/src/avm2/globals/Error.as

```diff
@@ -1,6 +1,8 @@
 package {
 	[Ruffle(InstanceAllocator)]
 	public dynamic class Error {
+		prototype.name = "Error"
+
 		public var name:String = "Error";
```

Comment:
Nit: 
```suggestion
		public var name:String;
```
(we set this property in the initializer)

Reply:
It's only set on the prototype, not as a field.